ui/pages/groups/[id].js

```diff
@@ -120,8 +123,30 @@ export default function GroupDetails() {
   ]
 
   // Don't allow deleting the last group
-  const hideRemoveGroupBtn =
-    !isAdmin || (infraAdmins?.length === 1 && adminGroups.includes(group?.id))
+  const showRemoveGroupBtn =
+    isAdmin && !(infraAdmins?.length === 1 && adminGroups.includes(group?.id))
+
+  function handleSelectedDeleteIds(deletedIds) {
+    setSelectedDeleteIds(deletedIds)
+  }
+
+  useEffect(() => {
+    setUsersData(
```

Comment:
Having a `useEffect` only to update another variable often means we can do this elsewhere in the code, or directly in the function here outside of the hook – could we remove it?